malleefowl/download.py

```diff
@@ -142,11 +142,20 @@ def download_files_from_thredds(url, recursive=False, cookie=None, monitor=None)
 
 
 def get_thredds_download_urls(url, cookie=None):
+    import threddsclient
```

Comment:
possibilité de faire un seul import dans le top du fichier?
`from threddsclient import get_thredds_urls`

Reply:
Well the dynamic import was already in the source when I start with it and I tought it was like that for performance issue... but now we are using this function pretty often so it might be a good move.

Reply:
not a dealbreaker, but better to import said function then import the whole package on each function call